android/src/oldarch/java/com/brentvatne/exoplayer/ReactExoplayerViewManager.java

```diff
@@ -441,4 +456,43 @@ private boolean startsWithValidScheme(String uriString) {
                 || lowerCaseUri.startsWith("file://")
                 || lowerCaseUri.startsWith("asset://");
     }
+
+    private @ResizeMode.Mode int convertToIntDef(String resizeModeOrdinalString) {
+        if (!TextUtils.isEmpty(resizeModeOrdinalString)) {
+            if (resizeModeOrdinalString.equals("none")) {
+                return ResizeMode.toResizeMode(ResizeMode.RESIZE_MODE_FIT);
+            } else if (resizeModeOrdinalString.equals("contain")) {
+                return ResizeMode.toResizeMode(ResizeMode.RESIZE_MODE_FIT);
+            } else if (resizeModeOrdinalString.equals("cover")) {
+                return ResizeMode.toResizeMode(ResizeMode.RESIZE_MODE_CENTER_CROP);
+            } else if (resizeModeOrdinalString.equals("stretch")) {
+                return ResizeMode.toResizeMode(ResizeMode.RESIZE_MODE_FILL);
+            }
+        }
+        return ResizeMode.RESIZE_MODE_FIT;
+    }
+
+    /**
+     * toStringMap converts a {@link ReadableMap} into a HashMap.
+     *
+     * @param readableMap The ReadableMap to be conveted.
+     * @return A HashMap containing the data that was in the ReadableMap.
+     * @see 'Adapted from https://github.com/artemyarulin/react-native-eval/blob/master/android/src/main/java/com/evaluator/react/ConversionUtil.java'
+     */
+    public static Map<String, String> toStringMap(@Nullable ReadableMap readableMap) {
```

Comment:
This function can be extracted to a helper function !

Reply:
Yeah it's true.
But we're not using it anywhere in the codebase.
So if you are okay, I will remove it.
But it's okay to move that function to another directory such as "helper"